web/app/view/ArrayListFilter.js

```diff
@@ -34,6 +34,13 @@ Ext.define('Traccar.view.ArrayListFilter', {
                         return true;
                     }
                 }
+            } else if (property.indexOf(',') !== -1) {
```

Comment:
What if it's space?

Reply:
Backend does not handle such case (space-separated).
But I can implement it in this custom component if we plan to use it somewhere else.

Reply:
I think it should be consistent. Your code below handles spaces.